RWKV-v4neo/config-example.yaml

```diff
@@ -83,12 +83,15 @@ trainer:
       filename: null
       
       # Save the top/last K checkpoints
-      save_top_k: 10
+      save_top_k: 5
       # Choose by the most recent checkpoints (time based)
```

Comment:
is this still time-based after change?

Reply:
Yea, and i learnt first hand how much 10 checkpoints can take in disk space haha. So toned it down abit

We could alternatively set to 3 (which is enough to recover from 'last checkpoint being corrupted' training runs)

Reply:
Sorry reread it, its now _step based, seems like the time value is not saved reliably on all steps, and it can crash the save process

Reply:
Then would you please update the comment line above?